tests/rip7560/test_send_failed.py

```diff
@@ -36,3 +38,75 @@ def test_eth_send_no_code_wrong_nonce(w3):
 
     ret = tx.send()
     assert_rpc_error(ret, "nonce too high", RPCErrorCode.INVALID_INPUT)
+
+
+def test_eth_send_account_validation_reverts(wallet_contract_rules, tx_7560):
+    tx_7560.sender = wallet_contract_rules.address
+    tx_7560.nonce = hex(2)
+    tx_7560.signature = to_prefixed_hex("revert-msg")
+
+    response = tx_7560.send()
+    assert_rpc_error(response, "validation phase reverted in contract account: on-chain revert message string", -32000)
+
+
+def test_eth_send_paymaster_validation_reverts(paymaster_contract_7560, tx_7560):
+    tx_7560.paymaster = paymaster_contract_7560.address
+    tx_7560.paymasterData = to_prefixed_hex("revert-msg")
+
+    response = tx_7560.send()
+
+    assert_rpc_error(
+        response,
+        "validation phase reverted in contract paymaster: on-chain revert message string",
```

Comment:
how can it return a "on-chain revert message string" ? this value is encoded inside Error(string), so someone did decode it